Mathlib/GroupTheory/NoncommCoprod.lean

```diff
@@ -46,6 +46,13 @@ def noncommCoprod (comm : ∀ m n, Commute (f m) (g n)) : M × N →ₙ* P where
   toFun mn := f mn.fst * g mn.snd
   map_mul' mn mn' := by simpa using (comm _ _).mul_mul_mul_comm _ _
 
+/-- Variant of `MulHom.noncommCoprod_apply` with the product written in the other direction` -/
+@[to_additive
+  "Variant of `AddHom.noncommCoprod_apply`, with the product written in the other direction"]
```

Comment:
```suggestion
  "Variant of `AddHom.noncommCoprod_apply`, with the sum written in the other direction"]
```
But maybe I am misunderstanding the statement.

Reply:
No, you're right.